packages/lesswrong/server/repos/BookmarksRepo.ts

```diff
@@ -49,7 +49,7 @@ class BookmarksRepo extends AbstractRepo<"Bookmarks"> {
         b."documentId",
         b."collectionName",
         c."postId",
-        c."directChildrenCount"
+        c."descendentCount"
```

Comment:
this is a backwards-breaking change that breaks old tabs

Reply:
I've left the existing field now

Reply:
doesn't seem to be here?